pkg/test/utils/kubernetes.go

```diff
@@ -774,3 +777,105 @@ func StartTestEnvironment() (env TestEnvironment, err error) {
 	env.DiscoveryClient, err = discovery.NewDiscoveryClientForConfig(env.Config)
 	return
 }
+
+// Kubectl runs a kubectl command (or plugin) with args.
+// args gets split on spaces (respecting quoted strings).
+func Kubectl(ctx context.Context, namespace string, args string, cwd string, stdout io.Writer, stderr io.Writer) error {
+	argSlice := []string{
+		"kubectl",
+	}
+
+	argSplit, err := shlex.Split(args)
+	if err != nil {
+		return err
+	}
+
+	argSlice = append(argSlice, argSplit...)
+	argSlice = append(argSlice, "--namespace", namespace)
```

Comment:
what happens if args already contain -n or --namespace? I guess the second one wins?

Reply:
Looks like it. I think the better behavior would be for me to check if it is already set and not set it if it is, though.